### PR TITLE
Math Magicians: Testing

### PR DESCRIPTION
**Pull request by manzitresor**
### Implemented Tests
- [ ] Test for **logics** 
    - `operate.js`
    - `calculate.js`
- [ ] Create unit tests for all React **components**
    - `NavBar.jsx`
    - `Home.jsx`
    - `Quote.jsx`
    - `Calculator.jsx`
    - `Buttons.jsx`

## Key Features:
 - Use Jest to write unit tests for logic
 - Use Jest snapshots to test the components
 - Use React Testing Library to simulate user interaction

### CODE REVIEW

File: src/__test__/operate.test.js
Comment (wayungi) [inline src/__test__/operate.test.js:1]: - [ ] Please write tests to cover all scenarios of the operate.js file.
   - Write tests for the modulo (%) 😸 
   - Write tests for the divide (÷) 😅

File: src/__test__/calculate.test.js
Comment (wayungi) [inline src/__test__/calculate.test.js:1]: - [ ] Please write tests to cover all scenarios of the calculate.js as is required in the project requirements. These scenarios include 💯 
 -  If there is an operation, update next 👍🏾 
 - If there is no operation, update next and clear the value 🤝 
 - '=' with no operation, nothing to do ✍🏾

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @manzitresor  & @manzitresor , 

image


**Congratulations on the good job done so far!** 🤝 

There are some issues that you still need to work on to go to the next project but you are almost there! 😄 

## Highlights ⚡ ⚡ 
- [ ] Good work writing some tests ✔️ 
- [ ]  Collaborated well ✔

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @manzitresor, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**